libs/agora/charts/src/lib/box-plot-chart/box-plot-chart.component.ts

```diff
@@ -50,6 +51,7 @@ export class BoxPlotComponent {
       return;
     }
 
+    if (this.heading !== '') this.plotHeight = '520px';
```

Comment:
Instead of overriding plotHeight on the existence of heading, perhaps you can add an overridable Input.  Something like this:

`@Input() plotHeight = '480px`